autoload/ale/completion.vim

```diff
@@ -148,6 +150,7 @@ let s:trigger_character_map = {
 \   'typescript': ['.', '''', '"'],
 \   'rust': ['.', '::'],
 \   'cpp': ['.', '::', '->'],
+\   'c': ['.', '->'],
```

Comment:
I think would be a good idea to honor the comment and update the ale.py accordingly.

Reply:
Done. Added it as part of the first patch